src/vsr/replica.zig

```diff
@@ -8466,7 +8587,8 @@ pub fn ReplicaType(
                 @panic("checkpoint diverged");
             }
 
-            if (candidate.view > self.view_durable()) {
+            // Ignoring newer view, we need to change view first when receiving a SV.
+            if (candidate.view > self.view_durable() and self.view < candidate.view) {
```

Comment:
Fix for the seeds `10266230273529621812` and `8842160722467235252`.
The replica's `view_durable` lags behind, but its `view` is already in the current view, so it ignores `SV`.